app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -59,6 +61,8 @@ public Product deleteProduct(Long id) {
 
     private Product findProduct(Long id) {
         return productRepository.findById(id)
-                .orElseThrow(() -> new ProductNotFoundException(id));
+                .orElseThrow(
+                    () -> new NotFoundException(Product.class.getSimpleName())
```

Comment:
클래스 이름을 예외에 실어 던져봐야 알 수 있는 것은 클래스 이름 뿐입니다. `id`를 남긴다면 디버깅하는 사람을 도와줄 수 있겠죠.

```suggestion
                    () -> new NotFoundException(String.format("id %d 에 해당하는 상품을 찾지 못했습니다.", id))
```

Reply:
아! 그렇군요!
수정하겠습니다.
감사합니다.